palette/palette.py

```diff
@@ -5,8 +5,15 @@
 
 @dataclass
 class Palette:
-    fence : Material = BasicMaterial(Block('oak_fence'))
-    frame : Material = BasicMaterial(Block('oak_log', is_log=True))
-    floor : Material = BasicMaterial(Block('oak_planks'))
-    roof : Material = BasicMaterial(Block('oak_log'))
-    wall : Material = BasicMaterial(Block('oak_planks'))
\ No newline at end of file
+    fence : Material = Block('oak_fence').material()
+    frame : Material = Block('oak_log', is_log=True).material()
+    floor : Material = Block('oak_planks').material()
+    roof : Material = Block('oak_log').material()
+    wall : Material = Block('oak_planks').material()
+    wall_stairs : Material = Block('oak_stairs', is_stairs=True).material()
+    wall_wall : Material = Block('cobblestone_wall').material() # listen we gotta figure out a better naming system
```

Comment:
A wall in minecraft terms behaves the same as a fence. They both could be classified under fence, or wall_fence if needed in for the wall generator in some form.